system_tests/tests/dg645.py

```diff
@@ -0,0 +1,232 @@
+import unittest
+
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import get_default_ioc_dir
+from utils.test_modes import TestModes
+from utils.testing import get_running_lewis_and_ioc, skip_if_recsim
+import time
+
+DEVICE_PREFIX = "DG645_01"
+
+IOCS = [
+    {
+        "name": DEVICE_PREFIX,
+        "directory": get_default_ioc_dir("DG645"),
+        "macros": {},
+        "emulator": "Dg645",
+    },
+]
+
+TEST_MODES = [TestModes.DEVSIM]

```

Comment:
Because Delaygen module does not support RECSIM and crashes in RECSIM mode. I could either
- add support for RECSIM to Delaygen
- copy records from Delaygen into DG645 support module and make them support RECSIM (currently 30 records that are tested)
I would like to hear opinion of next reviewer if it's worth doing